run_test.sh

```diff
@@ -14,6 +14,10 @@ if [[ ! -f $VENV ]]; then
     then
         echo "Installing dbt-sqlserver"
         pip install dbt-sqlserver --upgrade --pre
+    elif [ $1 == 'synapse' ]
+    then
+        echo "Installing dbt-synapse"
+        pip install dbt-synapse --upgrade --pre
```

Comment:
I think the generic logic below should handle this case, right? If `$1` is `synapse`, then
```
pip install dbt-$1 --upgrade --pre
```
becomes
```
pip install dbt-synapse --upgrade --pre
```

Reply:
Ah! Good catch you are absolutely right. This would render this PR moot in terms of my changes.